hal/armv7m/stm32/l4/Makefile

```diff
@@ -0,0 +1,28 @@
+#
+# Makefile for Phoenix-RTOS loader (ARMv7M HAL stm32l4x6)
+#
+# Copyright 2021 Phoenix Systems
+#
+# %LICENSE%
+#
+
+INIT_RAM = 08000000
+INIT_FLASH = 08000000
+
+BSS=-Tbss=20010000
+LDFLAGS+=--defsym _plo_bss=0x20010000
+
+LDFLAGS:=$(filter-out -Tbss% , $(LDFLAGS))
+LDFLAGS:=$(filter-out -Tdata% , $(LDFLAGS))
+
+CFLAGS:=$(filter-out -mfloat-abi% , $(CFLAGS))
+CFLAGS+= -mfloat-abi=soft
+
+GCCLIB := $(shell $(CC) $(CFLAGS) -print-libgcc-file-name)
+
+PLO_COMMANDS = alias app call console copy dump echo go help kernel kernelimg map mpu phfs script wait
```

Comment:
I am not a user of `stm32` but maybe to save the memory, it is worth to remove `kernel` command.

Reply:
Done